Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbUpgrader13To14.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Central Repository
+ *
+ * Copyright 2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.centralrepository.datamodel;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.sleuthkit.datamodel.CaseDbSchemaVersionNumber;
+
+
+/**
+ * This class updates CR schema to 1.4
+ * 
+ *  New correlation types for accounts are added, as well as some accounts related new tables are added in this version.
+ * 
+ */
+public class CentralRepoDbUpgrader13To14 implements CentralRepoDbUpgrader {
+
+    @Override
+    public CaseDbSchemaVersionNumber upgradeSchema(CaseDbSchemaVersionNumber dbSchemaVersion, Connection connection) throws CentralRepoException, SQLException {
+
+        if (dbSchemaVersion.compareTo(new CaseDbSchemaVersionNumber(1, 4)) < 0) {
+
+            try (Statement statement = connection.createStatement();) {
+
+                CentralRepoPlatforms selectedPlatform = CentralRepoPlatforms.getSelectedPlatform();
+
+                // Create account_types and accounts tables which are referred by X_instances tables
```

Comment:
I think it would be more accurate to say "...referenced by the X_acct_instances tables."